### PR TITLE
build: use GH Actions for upgrade testing

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-4.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -137,6 +129,89 @@ jobs:
...
+          composer remove burdamagazinorg/thunder
+          composer require "thunder/thunder-distribution:*" drupal/core:~8.9.0 --no-update
+          composer update
+          composer require "thunder/thunder_testing_demo:3.x-dev" "drupal/media_entity:^2.0-beta4" "drupal/video_embed_field:^2.2" "drupal/media_entity_image" "drupal/riddle_marketplace:^3.0-beta2"
+          vendor/bin/drush updb -y
+          vendor/bin/drush en vem_migrate_oembed -y
+          vendor/bin/drush vem:migrate_oembed
+          vendor/bin/drush cr
+          cd docroot
+          php core/scripts/db-tools.php dump-database-d8-mysql > dump.sql
```
Comment (ol0lll) [inline .github/workflows/test.yml]: I guess this is not a sql file, also where is it used?

**Review by ol0lll — COMMENTED** [review]
I think the removal of travis integration config and scripts files justifies a separate pr.

File: .github/workflows/test.yml (hunk 1 above)
Comment (chrfritsch) [reply .github/workflows/test.yml]: Right, it's a php file and it's used here 
https://github.com/thunder/drupal-testing/blob/master/lib/stages/run_tests.sh#L23

**Review by ol0lll — APPROVED** [review]

**Comment by chrfritsch** [comment]
Oh, eine Mietz 😍

**Pull request merged**